src/mod/server.mod/servmsg.c

```diff
@@ -20,6 +20,8 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  */
 
+#undef answer /* before resolv.h because it could collide with src/mod/module.h
```

Comment:
guard with an "#ifdef answer" or similar

Reply:
is there proof we need a guard around undef? i did a small test that shows we can undef arbitrary strings that are not defined:
```
$ cat test.c
#undef tommyknockers23
int main() {
  return 0;
}
$ gcc test.c -o test
```
if you still want me to guard it, let me know and ill do it.